components/user/UserTabs.js

```diff
@@ -30,6 +31,10 @@ export default function UserTabs({ tabs, setTabs }) {
             <option key={tab.name}>{tab.name}</option>
           ))}
         </select>
+        }
+        {tabs.length==1 && 
+          <div class="border-b border-gray-200"><nav class="-mb-px flex" style={{justifyContent: "center"}}aria-label="Tabs"><Link rel="noreferrer" class="border-indigo-500 text-indigo-600 w-1/4 py-4 px-1 text-center border-b-2 font-medium text-sm" aria-current="page" href="/dhruvilmehta#">My Links ({tabs[0].total})</Link></nav></div>
```

Comment:
not sure why this has your hardcoded user name in it? I think the extra nav is not required

Reply:
Yes, very sorry for that mistake.